endpoints.json

```diff
@@ -165,13 +165,15 @@
                     "team_id": 1,
                     "division_id": 2,
                     "team_name": "Leicester Wolves",
-                    "points": 45
+                    "team_start_time": "1500",
```

Comment:
if both teams are sharing the same venue, then the start times need to be different. Probably need to leave like 3 hours gap to be safe